viewer/packages/utilities/src/worldToViewport.ts

```diff
@@ -71,5 +71,15 @@ export function worldToViewportCoordinates(
  * Converts a pixel coordinate to normalized device coordinate (in range [-1, 1])
  */
 export function pixelToNormalizedDeviceCoordinates(x: number, y: number, width: number, height: number): THREE.Vector2 {
-  return new THREE.Vector2((x / width) * 2 - 1, (y / height) * -2 + 1);
+  return new THREE.Vector2((x / width) * 2 - 1, -(y / height) * 2 + 1);
+}
+
+export function getPixelCoordinatesToNormalized(
```

Comment:
It seems to me that this is mostly the same as the function above. Could we make use of `pixelToNormalizedDeviceCoordinates` by calling it from inside this function?

Reply:
Also - the name is a bit confusing, I think I'd prefer something like `getNormalizedPixelCoordinates`instead.

Reply:
Yes. The first one would be removed in the next PR. Renaming the othewr ok

Reply:
314413c300575b6f80c3c681266750399617bf9c